### ISSUE TITLE
node wiki update?

### ISSUE THREAD

**Issue by jayich** [labrad/pylabrad#178]
We are having trouble with our node when on commit 78939f5.  I am wondering if the wiki needs an update here.  From what I gather the command 

```bash
twistd -n labradnode
```

is outdated.

**Comment by maffoo** [comment]
I've updated the wiki page for the node to include the latest info. Please take a look and let me know if you see any other issues. Sorry for the confusion, but thanks for pointing out the discrepancy in the docs.

**Comment by jayich** [comment]
When we run the command

```
python -m larad.node
```

We are getting an error

```
NodeServer instance has no attribrute 'runners'
```

I think this is the tip of the iceberg, as there is some other behavior we don't understand.

**Comment by jayich** [comment]
I have a feeling we are missing something very basic, like not specifying a path to labrad.node.

**Comment by ejeffrey** [comment]
You you have a full stack trace?  self.runners is initialized in
NodeServer.initServer, so something must be happening before that, or
initServer isn't being called.  My guess would be that for some reason the
server isn't starting up properly and stopServer() is being called even
though initServer() never succeeded.  stopServer tries to shut down all
running services.

**Comment by ejeffrey** [comment]
OK, I have reproduced this, hope to have it figured out shortly.

**Comment by ejeffrey** [comment]
Hi andrew,

I just did a history search, and this was introduced with https://github.com/labrad/pylabrad/commit/53dc69b7de08a40a375f74d7c719d3dbea498e58 which added TLS support and was supposed to be fixed in https://github.com/labrad/pylabrad/commit/7393b36758e36d86203775cb67b3f1aad7c9a8ec, but apparently only partially worked.  As a workaround, you can set the environment variable LABRAD_TLS=no.  The default, LABRAD_TLS=starttls is the only mode that has the problem.

**Comment by jayich** [comment]
Great, this works:  LABRAD_TLS=off.  Thank you Evan.

**Comment by ejeffrey** [comment]
So I have tracked this down to the version of twisted.  crypto.py uses calls from twisted.internet.ssl that are only available in twisted 14.0.0 and later.  The version of twisted that is shipped with the current release of ubuntu LTS is twisted 13.2.0.  I will have to get @maffoo to comment on whether this is necessary.  It also looks like the twisted.internet.ssl module is not the recommended way to use SSL in twisted, but may be the closest analog to the way we are handling normal connections (which is one of the most archaic APIs in twisted)

**Comment by maffoo** [comment]
Good find, @ejeffrey. See also #173 about the lack of good error messages when the node fails to start. I've added some notes in the wiki about configuring the node, so closing this for now. We can discuss twisted versions in #179.
